serverless/develop/serverless-autoscaling-developer.adoc

```diff
@@ -0,0 +1,98 @@
+[id="serverless-autoscaling-developer"]
+= Autoscaling
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-autoscaling-developer
+
+toc::[]
+
+Knative Serving provides automatic scaling, or _autoscaling_, for applications to match incoming demand. For example, if an application is receiving no traffic, and scale-to-zero is enabled, Knative Serving scales the application down to zero replicas. If scale-to-zero is disabled, the application is scaled down to the xref:../../serverless/develop/serverless-autoscaling-developer.adoc#serverless-autoscaling-developer-minscale[minimum number of replicas specified for applications on the cluster]. Replicas can also be scaled up to meet demand if traffic to the application increases.
+
+If Knative autoscaling is enabled for your cluster, you can configure concurrency and scale bounds for your application.
+
+[NOTE]
+====
+Any limits or targets set in the revision template are measured against a single instance of your application. For example, setting the `target` annotation to `50` configures the autoscaler to scale the application so that each revision handles 50 requests at a time.
```

Comment:
The first sentence is passive voice. This one may be okay, but I am unsure.

Reply:
I think it's OK as is, this is talking about how Knative works - I think getting into individual autoscaling components in that sentence would be more confusing for readers.